src/transformers/configuration_blenderbot.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# coding=utf-8
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the;
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# LICENSE file in the root directory of this source tree.
+from .configuration_bart import BartConfig
+
+
+BLENDERBOT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "facebook/blenderbot-3B": "https://cdn.huggingface.co/facebook/blenderbot-3B/config.json",
+    "facebook/blenderbot-90M": "https://cdn.huggingface.co/facebook/blenderbot-/config.json",
+}
+
+
+class BlenderbotConfig(BartConfig):
+    """
+    This is the configuration class to store the configuration of a :class:`~transformers.BlenderbotForConditionalGeneration`.
+    Instantiating a configuration with the defaults will yield a similar configuration to that of
+    the `blenderbot <https://huggingface.co/blenderbot>`__ architecture.
+
+    Configuration objects inherit from  :class:`~transformers.BartConfig` and can be used
+    to control the model outputs. Read the documentation from  :class:`~transformers.BartConfig`
+    for more information. The
+
+    Args:
```

Comment:
This docstrings should be made better to conform to the ones of all other configurations, taking for instance `configuration_bert.py` as an example:
- `string` and `boolean` should become `str` and `bool`
- All arguments are optional, so this should be spelled out with an \`optional\`
- All arguments should have a description on a new indented line that contains no abbreviations (this is documentation, not code comment), begins with a capital and end with a period.